lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -637,13 +691,13 @@ protected function prepare_links( $post ) {
 		// Entity meta.
 		$links = array(
 			'self'       => array(
-				'href' => rest_url( $this->namespace . '/' . $this->parent_base . '/' . $post->post_parent . '/' . $this->rest_base . '/' . $post->ID ),
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent . '/font-faces/' . $post->ID ),
 			),
 			'collection' => array(
-				'href' => rest_url( $this->namespace . '/' . $this->parent_base . '/' . $post->post_parent . '/' . $this->rest_base ),
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent . '/font-faces' ),
 			),
 			'parent'     => array(
-				'href' => rest_url( $this->namespace . '/' . $this->parent_base . '/' . $post->post_parent ),
+				'href' => rest_url( $this->namespace . '/font-families/' . $post->post_parent ),
 			),
```

Comment:
parent_base is no longer being set by the constructor, so manually using `font-families` string for the links here.